### PR TITLE
SQL diagnostic - Tony Leo

### PR DESCRIPTION
**Pull request by antleo1995**
Struggled with this one. Still not sure why I had to make a primary key
as I was pretty sure this was done for us. So, that blew up on my tests.

Did the best I could but wasn't sure if I was to be using postgresql
or just plain ole SQL.

Pretty sure most of these are wrong.

### CODE REVIEW

File: diagnostic.sql [hunk 1]
```diff
@@ -5,24 +5,36 @@ CREATE DATABASE "sql-diagnostic";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname, given_name, gender, height, weight, born_on, eye_color, hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
--- your code here
+ CREATE TABLE licensees(
+   ID int PRIMARY KEY,
```
Comment (benjimelito) [inline diagnostic.sql:9]: I believe you want `serial` here rather than `int`

File: diagnostic.sql [hunk 2]
```diff
@@ -5,24 +5,36 @@ CREATE DATABASE "sql-diagnostic";
...
 
--- your code here
+UPDATE licensees SET height - 1 WHERE given_name = 'Marilynn' AND surname = 'Escobar';
+UPDATE licensees SET height - 1 WHERE given_name = 'Chris' AND surname = 'Waley';
 
 -- Question 5. Remove Dylan Rich and Teresita Myers from the table of licensees.
 
--- your code here
+DELETE FROM licensees WHERE given_name = 'Dylan' AND surname = 'Rich';
+DELETE FROM licensees WHERE given_name = 'Teresita' AND surname = 'Myers';
```
Comment (benjimelito) [inline diagnostic.sql:40]: You can actually do this query and the one above in one statement, like so:
```
DELETE FROM licensees
WHERE surname = 'Rich'
  AND given_name = 'Dylan'
OR surname = 'Myers'
  AND given_name = 'Teresita';
```

**Comment by benjimelito** [comment]
So the database does create a primary key for you, but in most cases you have to tell it to do so. No worries on not being 100% comfortable with this material, your answers look pretty good for the most part. 3/5

File: diagnostic.sql (hunk 1 above)
Comment (antleo1995) [reply diagnostic.sql:9]: Yeah, saw this after the fact. The time crunch means I just kind of shoot from the hip and then fix what I can if I have time.

**Pull request closed without merging**